webserver.py

```diff
@@ -127,7 +245,7 @@ def get_admin_data():
 
 def make_response(game_state, message=None, recipes=None):
     response = {
-        'state': game_state
+        'state': game_state.data
```

Comment:
rather than directly access the data, put it behind a method, "get_client_data"

Down the road, this will let us specify exactly which parts of the game state get sent to the client.  For instance, there's not a good reason to send the password back, ever.  So if you want to send all the data except the `password` field, this would be a good place.